astroquery/desi/core.py

```diff
@@ -0,0 +1,112 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import urllib.error
+import requests
+import warnings
+
+import pyvo as vo
+import numpy as np
+import astropy.coordinates as coord
+
+from astropy import units as u
+from astropy.units import Quantity
```

Comment:
This import is unused, triggering a code style error: https://github.com/astropy/astroquery/actions/runs/3327073137/jobs/5501457364#step:5:19
```suggestion
```